index.html

```diff
@@ -190,7 +227,7 @@ <h1 class="contact-title">Contact me</h1>
             <br>
             <input type="email" name="email" class="area email" placeholder="Email" required>
             <br>
-            <textarea name="message" class="message" maxlength="500" placeholder="Write your message here" required></textarea>
+            <textarea name="message" class="area message" maxlength="500" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] There is just one issue in your mobile version that I am sure you can fix it 💻. Please kindly increase the height of textarea which helps users send you more messages when they want to contact you 🤗🍀

#### Your design
![image](https://user-images.githubusercontent.com/69011963/150648258-9b36b847-5739-42a3-8a49-efc2fbe0e088.png)

#### Figma design
![image](https://user-images.githubusercontent.com/69011963/150648288-a892fc80-a084-4be2-949c-00e34951b16a.png)